benchmarks/distributed/rpc/parameter_server/servers/ParameterServerBase.py

```diff
@@ -0,0 +1,101 @@
+import functools
+import time
+from abc import ABC, abstractmethod
+
+import torch
+from metrics.MetricsLogger import MetricsLogger
+
+
+class ParameterServerBase(ABC):
+
+    PARAMETER_SERVER_BATCH_METRIC = "parameter_server_batch_metric"
+    PARAMETER_SERVER_STRAGGLER_METRIC = "parameter_server_straggler_metric"
+    BP_LOC_STRAGGLER = "bp_location_staggler"
```

Comment:
curious, what does "location" in the name mean?

Reply:
I need to update this constant to be `PARAM_LOC_STRAGGLER. 

ddp_communication_hook processes gradient buckets. By flattening and combining all the buckets, you can obtain a single list of parameters. The location for a bucket being processed is the starting parameter number from zero. 

> curious, what does "location" in the name mean?

location + 1 = parameter number of the first parameter in the bucket

The location is used for keys to record metrics.

Reply:
I will update `BP_LOC_STRAGGLER` to `PARAM_INDEX_STRAGGLER`